### PR TITLE
feat: Implement Places Autocomplete for location of events

### LINKED ISSUE #1736: Implement Places Autocomplete for location of events
**Describe the feature you'd like**
Implement Places Autocomplete for location of events.

**Would you like to work on the issue?**
- [x] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by ShridharGoel**
Fixes #1736 and #1689 

GIF for the change:

ezgif com-video-to-gif (89)

### CODE REVIEW

**Comment by iamareebjamal** [comment]
PRF

File: app/src/main/res/layout/event_details_step_one.xml [hunk 1]
```diff
@@ -198,96 +198,40 @@
...
+                        android:textSize="@dimen/text_size_normal"
+                        android:textStyle="bold" />
+
+                    <androidx.cardview.widget.CardView
                         android:layout_width="match_parent"
                         android:layout_height="wrap_content"
-                        android:padding="@dimen/spacing_extra_small">
+                        android:layout_margin="5dp"
+                        android:layout_marginTop="5dp"
+                        app:cardCornerRadius="4dp">
```
Comment (anhanh11001) [inline app/src/main/res/layout/event_details_step_one.xml]: extract resources here

Comment (ShridharGoel) [reply app/src/main/res/layout/event_details_step_one.xml]: Done.

File: app/build.gradle [hunk 2]
```diff
@@ -127,8 +127,8 @@ dependencies {
     //implementation "androidx.media:media:1.1.0-alpha04" // GradleIncompatible Workaround
     //implementation "androidx.legacy:legacy-support-v4:1.0.0" // GradleIncompatible Workaround
     annotationProcessor 'androidx.annotation:annotation:1.1.0' // Required for Glide
-    playStoreImplementation "com.google.android.gms:play-services-vision:${versions.play_services}"
-    playStoreImplementation "com.google.android.gms:play-services-places:${versions.play_services}"
+    playStoreImplementation "com.google.android.gms:play-services-vision:${versions.play_services_vision}"
+    implementation "com.google.android.libraries.places:places:${versions.places}"
```
Comment (iamareebjamal) [inline app/build.gradle]: Why?

Comment (ShridharGoel) [reply app/build.gradle]: Updated.

**Pull request merged**